### DISCUSSION TITLE
How do I pass tenant level configuration items into simple-list.html?

### DISCUSSION THREAD

**Discussion by catmanjan** [microsoft-search/pnp-modern-search#2595] [Q&A] [answered]
I need to be able to set a few configurable URLs once per tenant and have that setting flow through to simple-list.html 

What's the preferred way to get config items to the front end with pnp search?

**Comment by wobba** [comment, marked as answer]
You need a custom component to read settings. Or you copy simple-list.html and hardcode your settings in it, then reference all web parts to your .html in s central place. https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/#custom
